package.json

```diff
@@ -16,7 +16,9 @@
     "npm": "^6.4.0",
     "npm-run-all": "^4.1.3",
     "react": "^16.4.2",
+    "react-date-picker": "^7.10.0",
```

Comment:
Use the same datepicker that we use all around the project "semantic-ui-calendar-react".
Examples on usage can be found from: https://github.com/Metatavu/famifarm-management/blob/fbe5e200722b9f4218ce4a12bba6ff46a65f6960/src/components/ReportDownload.tsx#L81